changelog/12162.feature.md

```diff
@@ -0,0 +1,7 @@
+Location data in form of latitude and longitude will be picked up by the twilio channel for whatsapp and later can be processed in the custom actions.
+
+You can receive location data from user from this channel you need to do the following.
+
+Make an intent name locationData and two entities/slots one for Latitude and one for Longitude respectively.
+
+Once user send the location message it will Trigger the locationData intent and slots will be set via entities.

```

Comment:
Changelog entry will be listed on the [changelog page](https://rasa.com/docs/rasa/changelog/). Therefore they should be short and don't need to expand on the process

```suggestion
Add support for Location data from Whatsapp on Twilio Channel
```